enterprise/views.py

```diff
@@ -758,9 +759,12 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
         except (AttributeError, ValueError):
             course_effort = ''
         else:
-            course_effort = self.context_data['effort_hours_text'].format(
-                hours=effort_hours
-            )
+            course_effort = ungettext(
+                '{hours} hour per week, per course',
+                '{hours} hours per week, per course',
+                effort_hours,
+            ).format(hours=effort_hours)
```

Comment:
@UmanShahzad, we could potentially do it as a lambda on `context_data`. The trick is that we need the numeric variable in question to do the `ungettext` call.